HollowKnightSettings.cs

```diff
@@ -72,6 +81,48 @@ public partial class HollowKnightSettings : UserControl {
             }
             UpdateSplits();
         }
+        public void btnEdit_Click(object sender, EventArgs e) {
+            for (int i = flowMain.Controls.Count - 1; i > 0; i--) {
+                if (flowMain.Controls[i].Contains((Control)sender)) {
+                    HollowKnightSplitSettings setting = (HollowKnightSplitSettings)((Button)sender).Parent;
+                    if (setting.cboName.Enabled) {
+                        disableEdit(setting);
+                    } else {
+                        enableEdit(setting);
+                    }
+                    break;
+                }
+            }
+        }
+        private void btnAddAbove_Click(object sender, EventArgs e) {
+            for (int i = flowMain.Controls.Count - 1; i > 0; i--) {
+                if (flowMain.Controls[i].Contains((Control)sender)) {
+                    HollowKnightSplitSettings setting = (HollowKnightSplitSettings)((Button)sender).Parent;
+                    int index = setting.Parent.Controls.GetChildIndex(setting);
+                    addSplitAtIndex(index);
+                }
+            }
+        }
+        private void btnAddBelow_Click(object sender, EventArgs e) {
+            for (int i = flowMain.Controls.Count - 1; i > 0; i--) {
+                if (flowMain.Controls[i].Contains((Control)sender)) {
+                    HollowKnightSplitSettings setting = (HollowKnightSplitSettings)((Button)sender).Parent;
```

Comment:
this code is repeated. if I had more energy I would look into abstracting it, but /shrug

Reply:
its 3 lines, abstracting that if its only used <5 times its a waste of effort innit
i'll check out all the thingies tomorrow when im awaketh once more and as an excuse to practice 😎